src/Integration.Vsix.UnitTests/SonarLintDaemonTests.cs

```diff
@@ -40,56 +61,55 @@ public void Run_Without_Install()
         [TestMethod]
         public void Stop_Without_Start_Has_No_Effect()
         {
+            daemon.IsRunning.Should().BeFalse(); // Sanity test
             daemon.Stop();
+            daemon.IsRunning.Should().BeFalse();
         }
 
         [TestMethod]
-        [Ignore]
         public void Install_Reinstall_Run()
         {
             daemon.Install();
-            Assert.AreEqual(1, Directory.EnumerateFiles(tempPath).Count());
-            Assert.AreEqual(1, Directory.EnumerateDirectories(storagePath).Count());
+            Directory.GetFiles(tempPath).Length.Should().Be(1);
+            Directory.GetDirectories(storagePath).Length.Should().Be(1);
             Assert.IsTrue(daemon.IsInstalled);
             Assert.IsFalse(daemon.IsRunning);
 
             daemon.Install();
-            Assert.AreEqual(1, Directory.EnumerateFiles(tempPath).Count());
-            Assert.AreEqual(1, Directory.EnumerateDirectories(storagePath).Count());
-            Assert.IsTrue(daemon.IsInstalled);
-            Assert.IsFalse(daemon.IsRunning);
+            Directory.GetFiles(tempPath).Length.Should().Be(1);
+            Directory.GetDirectories(storagePath).Length.Should().Be(1);
+            daemon.IsInstalled.Should().BeTrue();
+            daemon.IsRunning.Should().BeFalse();
 
             daemon.Start();
-            Assert.IsTrue(daemon.IsInstalled);
-            Assert.IsTrue(daemon.IsRunning);
+            daemon.IsInstalled.Should().BeTrue();
+            daemon.IsRunning.Should().BeTrue();
             daemon.Stop();
-            Assert.IsFalse(daemon.IsRunning);
+            daemon.IsRunning.Should().BeFalse();
         }
 
         [TestCleanup]
         public void CleanUp()
         {
-            //ForceDeleteDirectory(tempPath);
-            //ForceDeleteDirectory(storagePath);
+            ForceDeleteDirectory(tempPath);
+            ForceDeleteDirectory(storagePath);
         }
 
         private static void ForceDeleteDirectory(string path)
         {
-            var files = Directory.GetFiles(path);
-            var directories = Directory.GetDirectories(path);
-
-            foreach (string file in files)
+            foreach (string file in Directory.EnumerateFiles(path))
             {
                 File.SetAttributes(file, FileAttributes.Normal);
                 File.Delete(file);
             }
 
-            foreach (string dir in directories)
+            foreach (string dir in Directory.EnumerateDirectories(path))
             {
                 ForceDeleteDirectory(dir);
             }
 
-            Directory.Delete(path, true);
+            Thread.Sleep(1);
```

Comment:
This fixes the file locking problem that made you comment out some lines last week?